openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>short comment on: why is this useful? what can it be used for?</p><ul><li>Maybe you only want to see quickly the dynamics of a system.</li><li>or want to pre equilibrate a given complex in order to use it for FEs later.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>